### ISSUE TITLE
Administative Organisations-Dashboard

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#278]
> [<img alt="datenbrei" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2018-11-27T06:56:48Z" title="Tuesday, November 27th 2018, 7:56:48 am +01:00">Nov 27, 2018</time>_
_Closed <time datetime="2020-02-18T20:30:42Z" title="Tuesday, February 18th 2020, 9:30:42 pm +01:00">Feb 18, 2020</time>_
---

As organisation support I'd like to have an overview about the status and profile of every organisation in order to keep control of the onboarding process of organisations and unlock a specific organisation's profile.

**Comment by Tirokk** [comment]
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-03-19T15:58:38Z" title="Tuesday, March 19th 2019, 4:58:38 pm +01:00">Mar 19, 2019</time>_

---


@datenbrei can you clarify this? I struggle to understand both the solution or the problem.

**Comment by Tirokk** [comment]
> [<img alt="stale[bot]" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/apps/stale) **Authored by [stale[bot]](https://github.com/apps/stale)**
_<time datetime="2019-11-19T22:53:22Z" title="Tuesday, November 19th 2019, 11:53:22 pm +01:00">Nov 19, 2019</time>_

---


This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

**Comment by Tirokk** [comment]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-20T19:09:59Z" title="Wednesday, November 20th 2019, 8:09:59 pm +01:00">Nov 20, 2019</time>_

---


still waiting for clarification... duplicate of https://github.com/Human-Connection/Human-Connection/issues/418 ??
blocked by https://github.com/Human-Connection/Human-Connection/issues/281

**Comment by Tirokk** [comment]
> [<img alt="stale[bot]" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/apps/stale) **Authored by [stale[bot]](https://github.com/apps/stale)**
_<time datetime="2020-01-19T19:41:59Z" title="Sunday, January 19th 2020, 8:41:59 pm +01:00">Jan 19, 2020</time>_

---


This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.
